iceoryx_posh/source/runtime/service_discovery.cpp

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2019 - 2021 by Robert Bosch GmbH. All rights reserved.
```

Comment:
Just because I liked the copyright discussion. Here we copied the copyright header just because it was used in the other file but we do not know if the code lines are actually provided by Bosch or not. Otherwise we are adding a copyright for code which may not originated at Bosch and this may has some weird or bad consequences.

Either we proof that the lines copied originated at Bosch or we just simplify this copyright thingy.

Here I suggest the Thursday meeting with the developers. We announce that we only keep the apache 2.0 header spdx identifier and name and thank all contributors on the website and readme.
This has no legal implications at all at ends this darn copyright thing once and for all. We should not let us dictate silly big company processes in this project.

If we insist on silly big company process I insist on a proof that the lines originates at Bosch by reviewing the line changes in the git history. Here we have to identify first who wrote the line in the beginning and if Bosch changed/adjusted the line was it significant enough - when they did not contribute it.